### PR TITLE
Test concepto empty

### PR DESCRIPTION
**Pull request by pachCode**
related #14

### CODE REVIEW

**Comment by matin** [comment]
@pachCode please move all tests to tests/ and VSR cassettes to tests/cassettes. See [arcusd](https://github.com/cuenca-mx/arcus-python) as a reference.

File: tests.py [hunk 1]
```diff
@@ -60,6 +60,13 @@ def test_create_orden(initialize_stpmex, get_order):
     assert orden._id == resp.id
 
 
+@vcr.use_cassette()
+def test_empty_concepto(initialize_stpmex,get_order):
+    orden = get_order
+    orden.conceptoPago=''
+    resp = orden.registra()
+    assert resp.descripcionError is not None
```
Comment (matin) [inline tests.py]: this should fail. The point is to write a failing test then fix the test.

Comment (pachCode) [reply tests.py]: De hecho verifica que rep tenga un error, o ya me perdí

Comment (matin) [reply tests.py]: like this: https://github.com/cuenca-mx/stpmex-python/pull/15/commits/110650f2ffa59e657d425df328a2c5d1cfaf25e9

File: test_stpmex.py [hunk 2]
```diff
@@ -60,6 +60,15 @@ def test_create_orden(initialize_stpmex, get_order):
     assert orden._id == resp.id
 
 
+@pytest.mark.xfail
```
[the remark this answers is not in the document]
Comment (pachCode) [reply test_stpmex.py]: Listo

**Pull request merged**